packages/main/src/plugin/extensions-catalog/extensions-catalog-api.ts

```diff
@@ -36,6 +36,7 @@ export interface CatalogExtension {
   extensionName: string;
   displayName: string;
   categories: string[];
+  keywords: string[];
```

Comment:
I'm not sure if we want to have these keywords mandatory, or optional?

Reply:
it's stored in the remote catalog so yes

but I would say that if omitted in the online catalog, default to []

so it's always here but it will return an empty array if omitted online